src/module/classes/GmScreenApplication.ts

```diff
@@ -308,7 +308,7 @@ export class GmScreenApplicationCommon extends Application {
       return;
     }
 
-    const entityUuid = `${data.pack ? `Compendium.${data.pack}` : data.type}.${data.id}`;
+    const entityUuid = `${data.uuid}`;
```

Comment:
```suggestion
    const entityUuid = data.uuid;
```
No need to convert this into a string with only itself inside

Reply:
Thanks! Wasn't sure about type safety here so I kept it this way. :)